ibr/ibr-sdk/class/Layer.js

```diff
@@ -0,0 +1,93 @@
+/**
+* Swap endianness of 32bit numbers.
+* @param {number} val 32 bit number to be swapped.
+* @return {number} 32bit number in swapped endianness.
+*/
+function swap32(val) {
```

Comment:
nit: this is defined elsewhere already right? we should consider moving this to a shared helper file soon